src/database/migrations/20240513082210-create-certificate-base-templates-table.js

```diff
@@ -32,6 +33,14 @@ module.exports = {
 			meta: {
 				type: Sequelize.JSON,
 			},
+			created_by: {
+				type: Sequelize.STRING,
+				allowNull: true,
```

Comment:
Can we mandate created_by and updated_up?

Reply:
updated created_by as not nullable. but updated_by can be null

Reply:
ok @priyanka-TL .